homeassistant/components/airzone/__init__.py

```diff
@@ -62,11 +65,28 @@ def get_zone_value(self, key):
         return value
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    if config_entry.version == 1:
+        new = {**config_entry.data}
+        new[CONF_ID] = DEFAULT_SYSTEM_ID
+
+        config_entry.version = 2
+        hass.config_entries.async_update_entry(config_entry, data=new)
+
+    _LOGGER.info("Migration to version %s successful", config_entry.version)
+
+    return True
```

Comment:
We don't need a migration for this (and thus do not need the version bump).

The version bump causes it to be not backward compatible (e.g., when a user downgrade Home Assistant). However, if this key is added, it still if backward compatible.

Additionally, we can add this key to the entry when missing without a migration (or default to it when getting it from the entry data).

Doing that, will remove the need for this migration.

Reply:
Thanks @frenck, you're totally right :).
Migration removed in 40a7154e671650656570917fa9f0b65aef501ae9